css/css-basics_button-styles/css/styles.css

```diff
@@ -29,20 +29,34 @@ button {
 
 /* Add your CSS styles below this line. Feel free to replace the comments with your code. Make sure not to put your code into the comments, because then nothing will happen. */
 
+button:hover{
```

Comment:
The space after the selector is missing. It is not mandatory but Sven said it should be there. In Visual Studio Code on Mac you can press Shift + Cmd + p to open the searchbar for orders and chose "format document". In my version it formats the document when saving. I think you can find the option to activate that as well.